style-guide.md

```diff
@@ -74,7 +74,7 @@ React elements are [immutable](https://en.wikipedia.org/wiki/Immutable_object).
 ✅ OK:
 
 ```md
-Los elementos de React son [inmutables](https://es.wikipedia.org/wiki/Objeto_inmutable).
+Les éléments de React sont [immuable](https://fr.wikipedia.org/wiki/Objet_immuable).
```

Comment:
This line should be in English even if the link could stay in French

Reply:
In the [translation guide example](https://www.gatsbyjs.org/contributing/gatsby-docs-translation-guide/#external-links) external links are in spanish

Reply:
I agree with that 👍